src/main/java/software/amazon/documentdb/jdbc/DocumentDbConnectionProperty.java

```diff
@@ -16,6 +16,7 @@
 
 package software.amazon.documentdb.jdbc;
 
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Isn't it a GPL license? Can we include it in our code?

Reply:
Sorry. Bad assumption. Researched the "best" NonNull solution but forgot to check the license. Good catch!

Reply:
Hmm. This is an often asked question ...

I believe we've researched this topic before and came to the conclusion the Annotation portion is MIT license and that is what we have in the [NOTICE](https://github.com/aws/amazon-documentdb-jdbc-driver/blob/5ec825ceb846606776367afeda66148b08439b92/NOTICE#L377) file.

https://groups.google.com/g/guava-discuss/c/wUmhhOP2Kpo?pli=1

https://checkerframework.org/manual/#license
https://raw.githubusercontent.com/typetools/checker-framework/master/LICENSE.txt
```
A few parts of the Checker Framework have more permissive licenses, notably
the parts that you might want to include with your own program.

 * The annotations and utility files are licensed under the MIT License.
   (The text of this license also appears below.)  This applies to
   checker-qual*.jar and checker-util.jar and all the files that appear in
   them, which is all files in checker-qual and checker-util directories.
   It also applies to the cleanroom implementations of
   third-party annotations (in checker/src/testannotations/,
   framework/src/main/java/org/jmlspecs/, and
   framework/src/main/java/com/google/).
```

Reply:
👍